wallets/electrum/src/main/java/bisq/wallets/electrum/notifications/ElectrumNotifyApi.java

```diff
@@ -36,22 +37,22 @@ public interface Listener {
         void onAddressStatusChanged(String address, String status);
     }
 
-    private static final CopyOnWriteArrayList<Listener> sListeners = new CopyOnWriteArrayList<>();
+    private static final List<Listener> listeners = new CopyOnWriteArrayList<>();
 
     @POST
     @Consumes("application/json")
     public Response notifyEndpoint(@Parameter(required = true) ElectrumNotifyRequest request) {
```

Comment:
why are you using swagger annotation here? Not needed if swagger is not used.
Also MediaType.APPLICATION_JSON could be used instead of the String "application/json"